pegasus/routes/poste_routes.rb

```diff
@@ -63,8 +63,21 @@
   template_params = JSON.parse(params[:params]) unless params[:params].to_s.empty?
   template_params ||= {}
 
+  recipients_csv = CSV.parse(params[:recipients_file][:tempfile].read, {headers: true})
+
```

Comment:
should there be an error if the csv is in the wrong format (for example, if there is no email column)